api/mfx_dispatch/linux/mfxloader.cpp

```diff
@@ -472,6 +482,16 @@ mfxStatus MFXVideoUSER_Load(mfxSession session, const mfxPluginUID *uid, mfxU32
 
 mfxStatus MFXVideoUSER_LoadByPath(mfxSession session, const mfxPluginUID *uid, mfxU32 version, const mfxChar *path, mfxU32 /*len*/)
 {
+  if (*uid == MFX_PLUGINID_HEVCD_SW ||
```

Comment:
create a (static inline) function which makes this check...

Reply:
Done